neuralmonkey/decoders/sequence_classifier.py

```diff
@@ -24,18 +24,49 @@ def __init__(self,
                  encoders: List[Any],
                  vocabulary: Vocabulary,
                  data_id: str,
-                 layers: Optional[List[int]]=None,
-                 activation: Callable[[tf.Tensor], tf.Tensor]=tf.tanh,
+                 layers: List[int]=None,
+                 activation: str='RELU',
                  dropout_keep_prob: float=0.5,
                  save_checkpoint: Optional[str]=None,
                  load_checkpoint: Optional[str]=None) -> None:
+        """Construct a new instance of the sequence classifier.
+
+        Args:
+            name: Name of the decoder. Should be unique accross all Neural
+                Monkey objects
+            encoders: Input encoders of the decoder
+            vocabulary: Target vocabulary
+            data_id: Target data series
+            layers: List defining structure of the NN. Ini example:
+                    layers=[100,20,5] ;creates classifier with hidden layers of
+                                       size 100, 20, 5 and one output layer
+                                       depending on the size of vocabulary
+            activation_fn: activation function used on the output of each
+                           hidden layer. Possible: 'none', 'relu', 'tanh',
+                           'sigmoid', 'elu'
+            dropout_keep_prob: Probability of keeping a value during dropout
+        """
         ModelPart.__init__(self, name, save_checkpoint, load_checkpoint)
 
         self.encoders = encoders
         self.vocabulary = vocabulary
         self.data_id = data_id
         self.layers = layers
-        self.activation = activation
+
+        if activation.lower() == 'none':
+            self.activation_fn = None
+        elif activation.lower() == 'relu':
+            self.activation_fn = tf.nn.relu
+        elif activation.lower() == 'elu':
+            self.activation_fn = tf.nn.elu
+        elif activation.lower() == 'tanh':
+            self.activation_fn = tf.tanh
+        elif activation.lower() == 'sigmoid':
+            self.activation_fn = tf.sigmoid
+        else:
+            raise ValueError(
+                "Unknown activation function: {}".format(activation))
```

Comment:
no a nebo udelat hybridni reseni (prepinac nebo callable) ale to se mi asi libi jeste mene nez nechani pouze callable :D